README.md

```diff
@@ -1,9 +1,12 @@
 ## Project Title: 
   Book Store
+## Summary: 
+This is a simple Book Store CMS project implemented with React.JS, Redux was used to set reduces and actions and API integretion through axios.
 ## Built With
 
 - React
 - Redux
```

Comment:
- [ ] Good job so far 👏 your README looks good but I would suggest that you update the `Getting started` section in order to include procedures to clone and run your project locally. This information is very important as it helps your fellow developers to get started with your project easily. so, please update it 😃